modyn/evaluator/internal/grpc/evaluator_grpc_servicer.py

```diff
@@ -16,6 +16,7 @@
     EvaluateModelRequest,
     EvaluateModelResponse,
     EvaluationData,
+    EvaluationNotStartReason,
```

Comment:
Can we rename this to `EvaluationNotStartedReason`?

Reply:
or something smaller, like `EvaluationCancelReason` or `EvaluationAbortReason` or `EvaluationFailureReason` or `EvaluationFailureReason `

That would be better to read.